providers/implementations/kdfs/build.info

```diff
@@ -0,0 +1,29 @@
+# We make separate GOAL variables for each algorithm, to make it easy to
+# switch each to the Legacy provider when needed.
+
+$TLS1_PRF_GOAL=../../libimplementations.a
+$HKDF_GOAL=../../libimplementations.a
+$KBKDF_GOAL=../../libimplementations.a
+$PBKDF2_GOAL=../../libimplementations.a
+$SSKDF_GOAL=../../libimplementations.a
+$SCRYPT_GOAL=../../libimplementations.a
+$SSHKDF_GOAL=../../libimplementations.a
+$X942KDF_GOAL=../../libimplementations.a
+
+SOURCE[$TLS1_PRF_GOAL]=tls1_prf.c
+
+SOURCE[$HKDF_GOAL]=hkdf.c
+
+SOURCE[$KBKDF_GOAL]=kbkdf.c
+
+SOURCE[$PBKDF2_GOAL]=pbkdf2.c
+# Extra code to satisfy the FIPS and non-FIPS separation.
+# When the PBKDF2 moves to legacy, this can be removed.
+SOURCE[../../libfips.a]=pbkdf2_fips.c
+SOURCE[../../libnonfips.a]=pbkdf2_fips.c
+
+SOURCE[$SSKDF_GOAL]=sskdf.c
+
+SOURCE[$SCRYPT_GOAL]=scrypt.c
+SOURCE[$SSHKDF_GOAL]=sshkdf.c
+SOURCE[$X942KDF_GOAL]=x942kdf.c
```

Comment:
Are none of the KDFs disablable? Should they be?

Reply:
```
: ; grep kdf Configure
: ; 
```

Reply:
I have no idea if they should be disablable.  With what I'm doing in #10124, I'm starting to think that the old style disabling may become outdated

Reply:
> starting to think that the old style disabling may become outdated

That is probably a good thing.  You'll need to catch all the old np-$CIPHER flags and fail the configuration with a message, but overall it still might be good.

Reply:
Either way we decide, that's for another PR

Reply:
Agreed - this is for another PR.